### ISSUE TITLE
New Large Examples (With walkthrough articles)

### ISSUE THREAD

**Issue by 200sc** [oakmound/oak#90]
We want to write more large examples like the platformer and top down shooter tutorials.

We'll store the code for these outside of the oak repo, but link to them from here.

Ideas:

 - Card game
 - Shootemup
 - Beatemup
 - Clicker

**Comment by Implausiblyfun** [comment]
Per conversation 8/1 there should be at least one example where a custom logger is used to override the dlog implementation.

Consider logrus or zap or zerolog.

Link to that example from the dlog docs to show how this can be done more easily.

**Comment by lolbinarycat** [comment]
I think I'll take a shot at the card game example.

**Comment by trifleneurotic** [comment]
Hello! this sounded fun as a first issue to tackle for Oak (shootemup), but it appears that there hasn't been any activity on this repository for a long time (and I acknowledge this issue is almost seven years old as well). Just wanted to comment to see if this project has been effectively abandoned or on its way to being archived?

**Comment by 200sc** [comment]
@trifleneurotic We aren't actively developing oak right now, and we'll meet tomorrow where I'll confirm this, but from my perspective oak isn't abandoned or planned to be archived. We'll review PRs and fix any raised bugs.

**Comment by Implausiblyfun** [comment]
Agreed. I've been lazy but happy to look over any prs that come up.

I still think that oak is quite functional and part of me is interested in seeing what updates we can pull given the core updates for x/shiny.
In other words havent done much recently but happy to check out or even collab on things in this repo!
